streams/src/main/java/org/apache/kafka/streams/KeyQueryMetadata.java

```diff
@@ -71,11 +75,40 @@ public HostInfo getActiveHost() {
      * Get the Store partition corresponding to the key.
      *
      * @return store partition number
+     * @deprecated Use {@link #partition()} instead.
      */
+    @Deprecated
     public int getPartition() {
         return partition;
     }
 
+    /**
+     * Get the Active streams instance for given key
```

Comment:
Nit: `[a]ctive` (i understand that it's just c&p but it seem wrong in the original JavaDoc, too.

`streams` -> `Kafka Streams`

Also, missing `.` at the end of the sentence.

Reply:
I'm with you, we should fix it, for the deprecated ones too.

Fixed.